maint/check-code-format.sh

```diff
@@ -37,23 +37,10 @@ else
     SED_FLAGS='-rn'
 fi
 
-# Find out python version. Use yapf only when in Python 3
-if PYTHON_VERSION=$(python -c 'import sys; print(sys.version_info[:][0])')
-then
-    if [[ "3" != "${PYTHON_VERSION}" ]]; then
-        echo "Python reports version number '${PYTHON_VERSION}' so will skip yapf formatting. Please use Python3"
-    fi
-else
-    bash_lib_print_error "Failed invocation of Python."
-    exit 1
-fi
-
 declare CLANG_FORMAT_BASENAME="clang-format-3.9"
 declare REQUIRED_CLANG_FORMAT_VERSION=3.9
-if [[ "3" == "${PYTHON_VERSION}" ]]; then
-    declare YAPF_FORMAT_BASENAME="yapf"
-    declare REQUIRED_YAPF_FORMAT_VERSION=0.26
-fi
+declare YAPF_FORMAT_BASENAME="yapf"
+declare REQUIRED_YAPF_FORMAT_VERSION=0.26
```

Comment:
the required version is declared but not enforced/checked

Reply:
Done.